src/config.js

```diff
@@ -33,12 +34,19 @@ const cdnProxyRegex =
     ? new RegExp(env['cdnProxyRegex'])
     : env['cdnProxyRegex'];
 
+const cdnSupportsCacheModifiedExtensions =
+  typeof env['cdnUrlSupportsCacheModifiedExtensions'] == 'string'
+    ? env['cdnUrlSupportsCacheModifiedExtensions'] !== 'false'
+    : env['cdnUrlSupportsCacheModifiedExtensions'] !== false;
```

Comment:
Can you explain why this is necessary?

Reply:
It's just a means of supporting `cdnUrlSupportsCacheModifiedExtensions` defined as either string or boolean.